### PR TITLE
build: prepare v0.18.1.0

### PR DESCRIPTION
**Pull request by selsta**

### CODE REVIEW

File: contrib/gitian/DOCKRUN.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ The dockrun.sh script will do everything to build the binaries. Just specify the
 version to build as its only argument, e.g.
 
 ```bash
-./dockrun.sh v0.17.3.0
+./dockrun.sh v0.18.1.0
```
Comment (hyc) [inline contrib/gitian/DOCKRUN.md:60]: Changes to this file should be made after https://github.com/monero-project/monero/pull/8442 is merged.

Comment (selsta) [reply contrib/gitian/DOCKRUN.md:60]: The PR you linked is against master branch and this one is against release branch.

Comment (hyc) [reply contrib/gitian/DOCKRUN.md:60]: Should we get seth to submit that against release branch then? Pointless to do these edits redundantly.

**Review by j-berman — APPROVED** [review]
Messaged @sethforprivacy about opening #8442 on the release branch too

**Comment by sethforprivacy** [comment]
I'll open #8442 against release branch today, sorry about that! Forgot that even doc changes need that.

**Comment by sethforprivacy** [comment]
Done: https://github.com/monero-project/monero/pull/8491

**Comment by sethforprivacy** [comment]
@selsta if you'd like, you can just drop your DOCKRUN.md changes and I'll bump mine to v0.18.1.0.

**Pull request merged**